ci_tools_atomic_dex/installer/windows/packages/com.komodoplatform.atomicdex/meta/package.xml

```diff
@@ -1,12 +1,12 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<Package>
-    <DisplayName>AtomicDEX Desktop</DisplayName>
-    <Description>Install AtomicDEX Desktop.</Description>
-    <Version>0.4.0</Version>
-    <ReleaseDate>2020-11-27</ReleaseDate>
-    <Licenses>
-        <License name="GNU General Public License v2.0" file="license.txt" />
-    </Licenses>
-	<Default>true</Default>
-	<Script>installscript.qs</Script>
-</Package>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
+<Package>
+    <DisplayName>AtomicDEX Desktop</DisplayName>
+    <Description>Install AtomicDEX Desktop.</Description>
+    <Version>0.4.0</Version>
+    <ReleaseDate>2020-11-27</ReleaseDate>
+    <Licenses>
+        <License name="GNU General Public License v2.0" file="license.txt" />
+    </Licenses>
+	<Default>true</Default>
+	<Script>installscript.qs</Script>
+</Package>
```

Comment:
I believe all those files can be deleted now since they are generated by CMake ?